rocketpool/api/node/deposit.go

```diff
@@ -92,13 +99,23 @@ func canNodeDeposit(c *cli.Context, amountWei *big.Int, minNodeFee float64, salt
 	var depositPoolBalance *big.Int
 
 	// Check credit balance
-	wg1.Go(func() error {
-		ethBalanceWei, err := node.GetNodeDepositCredit(rp, nodeAccount.Address, nil)
-		if err == nil {
-			response.CreditBalance = ethBalanceWei
-		}
-		return err
-	})
+	if isHoustonDeployed {
+		wg1.Go(func() error {
+			ethBalanceWei, err := node.GetNodeCreditAndBalance(rp, nodeAccount.Address, nil)
```

Comment:
I think this probably wants to be `GetNodeUsableCreditAndBalance` for deposit since getting the total amount doesn't help you if you can't use it?